pywwt/layers.py

```diff
@@ -162,11 +170,23 @@ class TableLayer(HasTraits):
     alt_unit = Any(help='The units to use for the altitude').tag(wwt='altUnit')
     alt_type = Unicode(help='The type of altitude').tag(wwt='altType')
 
-    cmap_att = Unicode(help='The column to use for the colormap').tag(wwt='colorMapColumn')
-
-    size_att = Unicode(help='The column to use for the size').tag(wwt='sizeColumn')
     size_scale = Float(10, help='The factor by which to scale the size of the points').tag(wwt='scaleFactor')
 
+    # NOTE: we deliberately don't link size_att to sizeColumn because we need to
+    # compute the sizes ourselves based on the min/max and then use the
+    # resulting column.
+    size_att = Unicode(help='The column to use for the size')
+    size_vmin = Float(None, allow_none=True)
+    size_vmax = Float(None, allow_none=True)
+
+    # NOTE: we deliberately don't link size_att to sizeColumn because we need to
```

Comment:
Should this comment reference `cmap_att` and something like `cmapColumn`?

Reply:
Fixed